components/nifosg/nifloader.cpp

```diff
@@ -1342,9 +1358,18 @@ namespace NifOsg
             }
             // unused by mw
             case Nif::RC_NiShadeProperty:
+            {
+                std::cerr << "RC_NiShadeProperty in " << mFilename  << " not currently used." << std::endl;
```

Comment:
The presence of these 3 properties in a file is not an error. As the comment above says, MW does not use those, so we shouldn't either. I specifically removed those warnings because people were complaining about the log spam with vanilla assets.